c/public/lib/source/t1read/t1read.c

```diff
@@ -55,6 +55,7 @@ static const char *keys[] =
 
 typedef unsigned short STI; /* String index */
 #define STI_UNDEF 0xffff    /* Undefined string index */
+#define STI_LIMIT 65000
```

Comment:
Is the limit 65000 or 65534?

Reply:
It's 65000 -- see section "10 String INDEX" of:
https://wwwimages2.adobe.com/content/dam/acom/en/devnet/font/pdfs/5176.CFF.pdf

Specifically:
> An SID is defined as a 2-byte unsigned number but only takes values in the range 0-64999, inclusive.  SID values 65000 and above are available for implementation use.